ui/src/app/components/multi-toast-banner.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { useState } from 'react';
+
+import colors from 'app/styles/colors';
+
+import { FlexColumn, FlexRow } from './flex';
+import { ArrowLeft, ArrowRight } from './icons';
+import { MultiToastMessage } from './multi-toast-message.model';
+import { ToastBanner } from './toast-banner';
+
+interface Props {
+  messages: MultiToastMessage[];
+  baseZIndex?: number;
```

Comment:
This is only here, because ToastBanner takes a zIndex. This allows us to pass the value through.